### DISCUSSION TITLE
Convert Indian Passport Size JPEG image file to Fix 40KB.

### DISCUSSION THREAD

**Discussion by sbind29893** [ImageMagick/ImageMagick#3197] [Help]
Hi Team,

Can you help in this case where i have set a Indian Passport size photograph to a fix size of 40KB using imagick.

File Input Format is JPEG

![Photo](https://user-images.githubusercontent.com/33458318/106774340-0915fd00-6668-11eb-8161-c8344fa0bd00.jpg)

**Comment by snibgo** [comment]
Your image is already less than 40kb.

Your can reduce it to, say, 15kb like this:
```
magick in.jpg -define jpeg:extent=15k out.jpg
```
The result is 14406 bytes.

**Reply by sbind29893** [reply]
Hi @snibgo 
Thanks for your reply, but i want this to be saved at a fixed size of 40 kb. The reason is the application on which i am trying to upload this images is accepting Photograph within a range of 20-50KB only.

**Reply by fmw42** [reply]
magick in.jpg -define jpeg:extent=40k out.jpg

**Comment by snibgo** [comment]
> Thanks for your reply, but i want this to be saved at a fixed size of 40 kb.

IM has no simple process that makes a JPG a fixed size of 40 kb.

Your input is smaller than that. You might add some noise, which would make it larger, then use "-define jpeg:extent=40k" to make it smaller.
